workers/loc.api/sync/data.inserter/halpers.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+const searchClosePrice = async (
+  dao,
+  {
+    auth,
+    symbol,
+    end,
+    id
+  }
+) => {
+  const trades = await dao.findInCollBy(
+    '_getTrades',
+    {
+      auth,
+      params: {
+        symbol,
+        end
+      }
+    }
+  )
+
+  if (
+    !Array.isArray(trades) ||
+    trades.length === 0
+  ) {
+    return null
+  }
+  if (
```

Comment:
Why is this if?

Reply:
For example, let's see the case of this `getPositionsHistory` respond:
```
[
...,
            {
                "symbol": "tBTCUSD",
                "status": "CLOSED",
                "amount": 0,
                "basePrice": 17869.84000001,
                "marginFunding": 0,
                "marginFundingType": 0,
                "pl": null,
                "plPerc": null,
                "liquidationPrice": null,
                "leverage": null,
                "placeholder": null,
                "id": 36222041,
                "mtsCreate": 1544631487000,
                "mtsUpdate": 1544637983000
            },
...
]
```
- for `"mtsUpdate": 1544637983000` requests trades from DB:
And in this case first row is closed trade (`orderID=1151367451`), rest no (`orderID=1151367423`)
![screenshot from 2019-02-13 10-51-02](https://user-images.githubusercontent.com/16489235/52699994-afe6c780-2f7f-11e9-914b-8978ac025b6a.png)

- but if do request trades for this `getPositionsHistory` respond (`"mtsUpdate": 1544623422000`):
```
[
...,
            {
                "symbol": "tBTCUSD",
                "status": "CLOSED",
                "amount": -0.88667444,
                "basePrice": 18787,
                "marginFunding": -4.31199977,
                "marginFundingType": 0,
                "pl": null,
                "plPerc": null,
                "liquidationPrice": null,
                "leverage": null,
                "placeholder": null,
                "id": 36222035,
                "mtsCreate": 1544606097000,
                "mtsUpdate": 1544623422000
            },
...
]
```
we do not have closed trade, and all `orderID=1151366801`
![screenshot from 2019-02-13 10-59-33](https://user-images.githubusercontent.com/16489235/52700032-c8ef7880-2f7f-11e9-9625-80f3d4e76d70.png)

It the way define availability closed trade
Let me know if you have a better way